### ISSUE TITLE
Thumbnails and icons

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/media-manager-improvement#567]
There have been various comments about the placement and usage of icons on the media thumbnails. I just came across this design pattern and thought it would be a good solution

![mediamanager](https://user-images.githubusercontent.com/1296369/42213333-5cac4872-7eb0-11e8-880e-2f8d1634bc0f.gif)

**Comment by ciar4n** [comment]
Rough draft of how that may look within the media manager context

### Default
![image](https://user-images.githubusercontent.com/2803503/42214384-50d2395a-7eb3-11e8-9d0c-267730d70a05.png)

### On hover
![image](https://user-images.githubusercontent.com/2803503/42214407-64f6f010-7eb3-11e8-87d1-77a877f5466f.png)

### Menu clicked
![image](https://user-images.githubusercontent.com/2803503/42214359-435289f6-7eb3-11e8-9191-936be1284af9.png)

**Comment by ciar4n** [comment]
The bigger 'actions' would allow us to add a title 👍 . Any particular complaints about the current icons?

**Comment by brianteeman** [comment]
Obviously I like it

**Comment by bahl24** [comment]
@franz-wohlkoenig @Anu1601CS Sorry for the delay, how does this look for starters?
![Screenshot from 2019-03-25 15-26-38](https://user-images.githubusercontent.com/36095178/54911117-c46d9680-4f13-11e9-8670-e7967d7a303c.png)

**Comment by ghost** [comment]
@bahl24 i don't know, i'm no dev, i try to hold Github clean :-) Can https://github.com/joomla/joomla-cms/issues/23968 be closed?

**Comment by bahl24** [comment]
@franz-wohlkoenig No, I am yet to make a PR. I wanted to ask how does the layout look.

**Comment by ghost** [comment]
> I wanted to ask how does the layout look

i don't know, but some who are working on the new Media manager may know (@laoneo ?)

**Comment by laoneo** [comment]
@bahl24 if you want to make a pr, please open one in the main repo. As development has shifted there.
